migrations/versions/480_add_briefs_table.py

```diff
@@ -0,0 +1,36 @@
+"""Add briefs table
+
+Revision ID: 480
+Revises: 470
+Create Date: 2016-01-25 15:45:50.363821
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '480'
+down_revision = '470'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+def upgrade():
```

Comment:
Yes, but not without squashing quite a lot of the whole commits together as well. I'll have a go at this in a branch and see how it looks.